umh.docs.umh.app/content/en/docs/whatsnew/0.2.md

```diff
@@ -211,3 +211,27 @@ Centralized all initial installation and continuous updating processes (Docker,
 - Updated to [0.3.0](https://github.com/united-manufacturing-hub/benthos-umh/releases/tag/v0.3.0)
   - Now supports connecting to Prosys OPC UA server, Microsoft OPC UA simulator (encrypted) & S7 via OPC UA.
   - Added encryption level auto discovery (This will attempt to connect with the highest encryption level first, and then step down if necessary).
+
```

Comment:
- Improved UI behavior when management console is offline
- Launch instance settings with the ability to delete an instance
- Double-clicking on the device in network view takes the user to the device information in component view